src/app/shared/learning-object/learning-object.component.ts

```diff
@@ -114,14 +114,14 @@ export class LearningObjectListingComponent implements OnInit, OnChanges {
   //Method for title case inside tool tip
   toTitleCase = function (str) {
```

Comment:
Is there any reason we're not using the Angular `titleCase` pipe or just CSS to render this?

Reply:
Yes using the pipe titlecase breaks the UI and I don't see a way to use css to capitalize whats inside the tooltip. I will include pictures of the error and what happens if I use the titlecase pipe 
![usingtitlecase1](https://user-images.githubusercontent.com/37280445/53030475-db165e80-3438-11e9-85ce-8e2d45cde625.PNG)
![usingtitlecase](https://user-images.githubusercontent.com/37280445/53030477-db165e80-3438-11e9-9335-1ac8c8be8a94.PNG)